### PR TITLE
doc: a note about networking in `examples/README.md`

### PR DESCRIPTION
**Pull request by kaspar030**
Closes https://github.com/future-proof-iot/RIOT-rs/issues/199.

*edit* not all examples are listed, those that are not will be moved to `tests/` once #189 is in.

### CODE REVIEW

File: examples/README.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+## Networking
+
+Some examples configure networking. By default, they will listen on a static
+IPv4 address. Be sure to configure the host computer end of the network
+accordingly.
+
+The default IPv4 address the test examples will configure is `10.42.0.61`.
+You can configure the other end like this:
+
+    $ ip address add 10.42.0.1/24 dev <DEVICE>
```
Comment (emmanuelsearch) [inline examples/README.md:25]: ip addr ?

File: examples/README.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    $ ip link set up dev <DEVICE>
+
+Replace `<DEVICE>` with the name of the used network device.
```
Comment (emmanuelsearch) [inline examples/README.md:28]: for real beginners, maybe add here something like 

- "to find out the name of your interface you can use a command such as ifconfig". 
- "to double-check that the address has indeed be added, you can use: ip addr show dev <DEVICE>"

File: examples/embassy-net-udp/README.md [hunk 3]
```diff
@@ -13,3 +13,5 @@ In this folder, run
 
 With the device USB cable connected, a USB ethernet device should pop up.
 RIOT-rs will reply to ping requests on 10.42.0.61.
+
```
Comment (emmanuelsearch) [inline examples/embassy-net-udp/README.md:16]: not sure if this is absolutely needed but is the ping -I <DEV> option needed here ? (If so add this precision for beginners?)

**Review by emmanuelsearch — APPROVED** [review]

File: examples/README.md (hunk 1 above)
Comment (kaspar030) [reply examples/README.md:25]: 'address' is actually the normal subcommand name. do you mean we should use a shorter version? The shortest would be just `ip a a ...`.

File: examples/embassy-net-udp/README.md (hunk 3 above)
Comment (kaspar030) [reply examples/embassy-net-udp/README.md:16]: `-I` shouldn't be needed?

File: examples/README.md (hunk 2 above)
Comment (kaspar030) [reply examples/README.md]: Added. And replaced "DEVICE" with "INTERFACE" where appropriate.

Comment (emmanuelsearch) [reply examples/README.md]: perfect

File: examples/README.md (hunk 1 above)
Comment (emmanuelsearch) [reply examples/README.md]: OK, let's keep it like that ;)

**Pull request merged**